var/spack/repos/builtin/packages/cbtf-argonavis-gui/package.py

```diff
@@ -51,50 +51,44 @@ class CbtfArgonavisGui(QMakePackage):
     url = "https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git"
 
     version('1.3.0.0', branch='1.3.0.0',
-            git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git')
+            git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git', preferred=True)
 
     version('develop', branch='master',
             git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git')
 
-    depends_on("cmake@3.0.2:", when='@develop', type='build')
-    depends_on("cmake@3.11.1", when='@1.3.0.0:', type='build')
-
-    # To specify ^elfutils@0.170 on the command line spack
-    # apparently needs/wants this dependency explicity here
-    # even though it is referenced downstream
-    depends_on("elf", type="link")
+    depends_on("cmake@3.0.2:", type='build')
 
     depends_on('qt@4.8.6:', when='@develop')
-    depends_on('qt@5.10.0', when='@1.3.0.0:')
+    depends_on('qt@5.10.0', when='@1.3.0.0:9999')
```

Comment:
I've always been confused by the dependencies in this package. It seems like the develop version supports older versions of dependencies than the latest release? It's usually the other way around...

Reply:
We wanted to go with a proven path for the release versions and let spack pick the latest version available for develop.  I could make the develop version start at a higher level qt version.

Reply:
@adamjstewart I will update the qt develop version asked for.